core/templates/dev/head/pages/admin/activities_tab/admin_dev_mode_activities_tab_directive.html

```diff
@@ -19,6 +19,29 @@
             ng-click="reloadAllExplorations()">
       Reload All Explorations
     </button>
+    <span class="row">
+      <h3>Generate dummy explorations</h3>
+      <label for="explorations to generate">Number of explorations to generate</label>
+      <input class="form-control form-inline" style="width:50%; margin-bottom: 5px"
+             ng-model="DummyExplorationCount"
+             placeholder="Explorations to generate" type="number">
+      <label for="explorations to publish">Number of explorations to publish</label>
+      <em ng-show="DummyExplorationCount"
+          style="font-size: smaller;">
+        (Publish <[DummyExplorationPublishCount + ' out of ' + DummyExplorationCount]> exploration)
+      </em>
+      <input class="form-control form-inline" style="width: 50%;"
+             ng-model="DummyExplorationPublishCount"
+             ng-init = "DummyExplorationPublishCount = 0"
```

Comment:
This seems like an abuse of ng-init. See the guidance [here](https://docs.angularjs.org/api/ng/directive/ngInit).